src/mixins/common-mixin.ts

```diff
@@ -60,9 +66,13 @@ export class CommonMixin extends Vue {
   entityTypeToCorpType (entityType: EntityTypes): CorpTypeCd {
     switch (entityType) {
       case EntityTypes.BC: return CorpTypeCd.BENEFIT_COMPANY
+      case EntityTypes.C: return CorpTypeCd.CONTINUE_IN
+      case EntityTypes.CBEN: return CorpTypeCd.BEN_CONTINUE_IN
       case EntityTypes.CC: return CorpTypeCd.BC_CCC
+      case EntityTypes.CCC: return CorpTypeCd.CCC_CONTINUE_IN
       case EntityTypes.CP: return CorpTypeCd.COOP
       case EntityTypes.CR: return CorpTypeCd.BC_COMPANY
+      case EntityTypes.CUL: return CorpTypeCd.ULC_CONTINUE_IN
```

Comment:
Same comment as above: I think these (and the 4 below) aren't used, but I added them for completeness.